### ISSUE TITLE
wrong service endpoint

### ISSUE THREAD

**Issue by Perfect-Web** [kubecost/cost-analyzer-helm-chart#565]
**Describe the bug**
```
            - name: COST_ANALYZER_SERVER_ENDPOINT
              value: {{ template "cost-analyzer.fullname" . }}.{{ .Release.Namespace  }}:9001
```
this is the template for the check

and this is the template for the service
```
metadata:
  name: {{ template "cost-analyzer.serviceName" . }}
```

they do not match

**To Reproduce**
deploy helm version 1.62.0

**Expected behavior**
to work after a fresh helm install

**Collect logs (please complete the following information):**
```Starting...
Error: Error: getaddrinfo ENOTFOUND cost-analyzer.cost-analyzer cost-analyzer.cost-analyzer:9003
    at Object.dispatchError (/node/node_modules/jsdom/lib/jsdom/living/xhr-utils.js:60:19)
    at Request.client.on.err (/node/node_modules/jsdom/lib/jsdom/living/xmlhttprequest.js:674:20)
    at emitOne (events.js:120:20)
    at Request.emit (events.js:210:7)
    at Request.onRequestError (/node/node_modules/request/request.js:877:8)
    at emitOne (events.js:115:13)
    at ClientRequest.emit (events.js:210:7)
    at Socket.socketErrorListener (_http_client.js:399:9)
    at emitOne (events.js:115:13)
    at Socket.emit (events.js:210:7) undefined```

**Comment by AjayTripathy** [comment]
Thanks for this report. This should be fixed in #540 -- expected in our next release.

**Comment by AjayTripathy** [comment]
This has been fixed-- let me know if you're still experiencing issues here.
